crane/crane.go

```diff
@@ -92,6 +94,19 @@ func checkDockerClient() {
 			printErrorf("Unsupported client version! Please upgrade to Docker %v or later.\n", intJoin(requiredDockerVersion, "."))
 		}
 	}
+	actualDockerVersion = versions
+}
+
+func validateDockerClientAbove(expectedVersions []int) bool {
```

Comment:
I am not sure this is really required, it implies a lot of glue code for little value. Besides, it should be a separate PR I think.